lib/realtime/api.ex

```diff
@@ -225,4 +230,32 @@ defmodule Realtime.Api do
     |> Map.put(:events_per_second_rolling, avg)
     |> Map.put(:events_per_second_now, current)
   end
+
+  defp maybe_invalidate_cache(%Ecto.Changeset{changes: changes, valid?: true, data: %{external_id: external_id}})
+       when changes != %{} do
+    Tenants.Cache.invalidate_tenant_cache(external_id)
+  end
+
+  defp maybe_invalidate_cache(_changeset), do: nil
+
+  defp maybe_trigger_disconnect(%Ecto.Changeset{data: %{external_id: external_id}} = changeset)
+       when requires_disconnect(changeset) do
+    Phoenix.PubSub.broadcast!(Realtime.PubSub, "realtime:operations:" <> external_id, :disconnect)
+  end
+
+  defp maybe_trigger_disconnect(_changeset), do: nil
+
+  defp maybe_restart_db_connection(%Ecto.Changeset{data: %{external_id: external_id}} = changeset)
+       when requires_restarting_db_connection(changeset) do
+    case Connect.lookup_or_start_connection(external_id) do
```

Comment:
Instead of `Connect.lookup_or_start_connection` should we use instead `Connect.whereis` so that we don't start Connect if the tenant is inactive to then shut it down? 🤔 WDYT?